### PR TITLE
Enable Giant Pincer Mutation

### LINKED ISSUE #76672: Cannot receive the mutation Giant Pincer from Crustacean Primer
### Describe the bug

No matter how long you wait, no matter how much Crustacean Primer you take, you will NEVER mutate Giant Pincer. You will get every other mutation in the tree and then the game will merely state "Your body tries to mutate, but it lacks a primer to do so and only contorts for a moment." Mutating Giant Pincer through the debug menu works fine. This was something I noticed in an earlier playthrough so made a test character to confirm this.

### Attach save file

[Worthington Springs-trimmed.tar.gz](https://github.com/user-attachments/files/17144651/Worthington.Springs-trimmed.tar.gz)

### Steps to reproduce

1. Make a player.
2. Mutate everything in the Crustacean tree except giant pincer
3. Take Crustacean Mutagenic Primer and Catalyst
4. After waiting, you will get messages in the log that your body tries to mutate, but lacks primer, even though it has primer and has a mutation it can still receive in that category (giant pincer).

### Expected behavior

I expect the Giant Pincer mutation to manifest, considering I don't have it and my body has both primer and catalyst.

### Screenshots

bug report 2
bug report 3
bug report 1

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.4894 (22H2)
- Game Version: cdda-experimental-2024-09-24-1915 7ee3ce7 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by TheMurderUnicorn**
#### Summary
Bugfixes "Enable Giant Pincer Mutation"

#### Purpose of change
#76672 pointed out that the Giant Pincer Mutation was unobtainable.  It was marked as not being obtainable except as a starting trait, but appeared in no character creation except for a mod.

#### Describe the solution
Changed "valid" to true, making it obtainable through regular mutation.

#### Describe alternatives you've considered
Leaving it in case I was wrong, but I looked into it and it's been unchanged since implemented, and I think it's just an oversight.

#### Testing
Enabling it allows it to be mutated, as expected.

#### Additional context
Did not appear in any starting scenarios/etc. that would enable it to be selected at start.  Trait prevented it from being obtained via catalyst.  Did not make sense.

Does not appear to have been disabled by WIP limb stuff (it's referenced there, but only as being replaced when losing an arm/etc).  Does not appear to have been modified since being added.  Does not appear obtainable.  Appears to be an oversight.

If I missed something, let me know.  There's some discussion in the linked issue.

closes #76672

### CODE REVIEW

**Comment by harakka** [comment]
Would you like to do this fix to 0.H-branch too by any chance once it's been approved for master?

**Comment by Karol1223** [comment]
Oh huh THAT'S why I couldn't get it through debug no matter what I did

**Comment by Kamejeir** [comment]
https://github.com/CleverRaven/Cataclysm-DDA/blob/b2e794000e8488db6dec84a296fb87bd969aa1c4/doc/MUTATIONS.md?plain=1#L99

"default: true" means you can just remove the field to have the same effect (I'm fairly sure, owing to the setting not being present on most mutations)

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -3454,7 +3454,7 @@
     "visibility": 10,
     "ugliness": 9,
     "description": "Instead of your right hand, you now have a giant claw resembling ones found on crabs.  It encumbers you greatly, but you can use it in melee combat to deal massive damage.",
-    "valid": false,
+    "valid": true,
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:3457]:
```suggestion
```

**Pull request merged**